site/cypress/e2e/Cypress-Feature/late_days_allowed.spec.js

```diff
@@ -55,12 +59,13 @@ describe('Test cases involving the late days allowed page', () => {
 
         it('correctly add a late day and then update it', () => {
             // add some late days for bitdiddle
-            cy.get('#user_id').type('bitdiddle').blur();
+            cy.get('#user_id').type('bitdiddle');
+            cy.get('#user_id').blur();
             cy.get('#datestamp').type('2021-01-01');
             cy.get('#user_id').click(); // dismiss the calendar view
             cy.get('#late_days').type('3');
             cy.get('input[type=submit]').click();
-            cy.wait(1000); // make sure the late day registered
+            cy.reload(); // make sure the late day registered
```

Comment:
Does this need to reload? I haven't tested it locally, but this (and the waits) seem a bit unnecessary

Reply:
Seems like it doesn't; fixed in https://github.com/Submitty/Submitty/pull/10562/commits/42e999e19e31ae96fa4f481aea4877f409139cf6